tests/Iterator/MessageManagerMessageIteratorTest.php

```diff
@@ -77,4 +80,28 @@ public function testLongForeach(): void
             }
         }
     }
+
+    public function testMessageConsumptionOrder(): void
+    {
+        $now = new \DateTime();
+        $olderDate = new \DateTime();
+        $olderDate->sub(new \DateInterval('PT1H'));
+        $olderMessage = new Message();
```

Comment:
can you rename it please?

`$olderMessage` -> '$oldMessage'
`$newerMessage` -> '$newMessage'